### DISCUSSION TITLE
Is it possible to use ZeroSSL to create SSL certificates? (This would enable SSL for IP addresses)

### DISCUSSION THREAD

**Discussion by eqoram** [pocketbase/pocketbase#3240] [Q&A]
Hello,

First of all thank you so much for this amazing project! I was wondering if pocketbase could use ZeroSSL for generating SSL certificates as this service seems to be able to support IP addresses as well as domains (https://help.zerossl.com/hc/en-us/articles/360060119973-Is-It-Possible-To-Generate-a-SSL-Certificate-for-an-IP-Address-). This would make it even more easy to host this service. :)

**Comment by ganigeorgiev** [comment]
No, at the moment it is not supported, at least not out of the box.

I have it my todo to consider it as an option but remains a low priority as the last time I checked ZeroSSL required an API key to work.
